sdk/src/evm/utils/customErrors.ts

```
@@ -0,0 +1,115 @@
+            data: reason as Hex,
+            abi,
+          });
+          console.error(`Error: ${decoded.errorName}, args: ${decoded.args}`);
+          processedError = {
+            ...rpcError,
+            errorName: decoded.errorName,
+            errorArgs: Array.from(decoded.args ?? []),
+          };
+        } catch (decodeErr) {
+          if (decodeErr instanceof AbiErrorSignatureNotFoundError) {
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)